sickrage/providers/GenericProvider.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, name):
             #'http://torcache.net/torrent/{torrent_hash}.torrent',
             'http://thetorrent.org/torrent/{torrent_hash}.torrent',
             'http://itorrents.org/torrent/{torrent_hash}.torrent',
+            'http://torrage.info/torrent.php?h={torrent_hash}.torrent',
```

Comment:
looks like in torrage we need to click a button to actual download. If we can't bypass, we cant add it